libs/coin-evm/src/api/gasTracker/types.ts

```diff
@@ -13,7 +14,7 @@ export type GasTrackerApi = {
   }) => Promise<GasOptions>;
 };
 
-type GasTrackerConfig = EthereumLikeInfo["gasTracker"];
+type GasTrackerConfig = EvmConfigInfo["gasTracker"];
```

Comment:
Same comment about where should be the source of truth about the type definition (maybe more in `EthereumLikeInfo`)